### PR TITLE
biocViews - howto-bcv

### PR DESCRIPTION
**Pull request by villafup**
@jwokaty Please review

### CODE REVIEW

**Comment by hpages** [comment]
Can we please stick to the standard .**R**md extension?

**Comment by LiNk-NY** [comment]
Hi Paul (?), 

Thanks for working on this!
Can you please keep the code close to the original + improvements? 

E.g.,
```r
library("biocViews")
library("Biobase")
```
should stay the same and avoid package installation commands i.e., `install.packages("biocViews")`

**Comment by jwokaty** [comment]
Thanks everyone for your input! @villafup is working on conversion project and this is his first PR, which I'm going to review and give feedback.

File: vignettes/HOWTO-BCV.rmd
Comment (jwokaty) [inline vignettes/HOWTO-BCV.rmd]: Should be `biocViews-HOWTO` rather than `Quick-start Guide`

File: vignettes/HOWTO-BCV.rmd
Comment (jwokaty) [inline vignettes/HOWTO-BCV.rmd]: As @LiNk-NY mentioned in https://github.com/Bioconductor/biocViews/pull/4#issuecomment-1150207341, we want to keep this close to the original as possible. We also don't typically `install` in vignettes. Assume that it's already installed and that we just need to use the `library` function.

File: vignettes/HOWTO-BCV.rmd
Comment (jwokaty) [inline vignettes/HOWTO-BCV.rmd]: Remove the surrounding spaces inside the 3 tick marks for this and other instances.

File: vignettes/HOWTO-BCV.rmd
Comment (jwokaty) [inline vignettes/HOWTO-BCV.rmd]: I would also remove all the extra spaces here and 134-135 just to get into a good habit.

**Review by jwokaty — CHANGES_REQUESTED** [review]
- [x] As @hpages mentioned, we want to keep the capitalization in the `.Rmd` extension: https://github.com/Bioconductor/biocViews/pull/4#issuecomment-1150195978.
- [x] When have the `.Rmd` file, we want to remove the corresponding `.Rnw` file. I would do this as a separate commit from your other changes with a precise commit message in case we ever needed to bring it back. To do this in git use `git rm` name_of_file.Rnw.
- [ ] You also need to bump the z number in the version (x.y.z) in the DESCRIPTION file.
- [ ] You probably should also add `BiocStyle` to the `Suggests` list in the DESCRIPTION file since it isn't there now.

Please address these changes, commit, and push your changes. When you `git push origin your-branch-name`, your new commits will be placed after your existing commits on this pull request.

**Comment by jwokaty** [comment]
@villafup This is in pretty good shape; however, I noticed that in this commit git didn't recognize that  `vignettes/HOWTO-BCV.rmd` got renamed to `vignettes/HOWTO-BCV.Rmd` so you need to do another commit where you delete the lowercase rmd file.

**Comment by LiNk-NY** [comment]
> [ ] You also need to bump the z number in the version (x.y.z) in the DESCRIPTION file.

It's not critical but I tend to leave this up to the maintainer esp. when the repository is in active maintenance (in order to avoid conflicts).

**Comment by jwokaty** [comment]
@LiNk-NY Thank you. I think we'll follow that recommendation from now on regarding bumping versions.

**Comment by jwokaty** [comment]
@villafup I think this is ready to merge. Great job!

**Pull request merged**
